packages/cfd/src/Components/cfd-account-card.tsx

```diff
@@ -25,10 +25,12 @@ const account_icons: { [key: string]: TAccountIconValues } = {
         financial_stp: 'IcMt5FinancialStpPlatform',
         cfd: 'IcMt5CfdPlatform',
     },
+    // TODO: Line 30, 31 and 32 should be removed after real released.
```

Comment:
@hamid-deriv For these todos, Do you mean they should be handled in the same PR or another one? 🤔

Reply:
There are two different releases for CFD accounts, First for Demo Accounts and then for Real Accounts. The second release will happen a week after the first one. So, I add these todos to remove/modify these lines before the second (and last) release.

Reply:
@hamid-deriv Got it 👌🏻 
I would suggest removing the todos in the code and instead leaving an unresolved comment on the PR itself, Todos can be missed easily and we might merge the PR without fixing them.

But it's just a suggestion 😅 That's how I usually do it 😬
You can ignore this comment 🙇🏻🌹

Reply:
The second release has a separate card and will have separate PR as well. :+1: